handbook/best-practice/system-admins.md

```diff
@@ -0,0 +1,27 @@
+# System Admins
+
+The System Admins for each of the systems DVELP uses are defined in the below
+table. It is the responsibility of the System Admin to make sure that only the relevant
+people have access to these systems and at the right permission level.
+
+We operate on the principle of least privilege, which means individuals should have
+the minimum level of privilege in order to be able to perform their role
+effectively.
+
+System Admins conduct a thorough review of the permissions once every calendar
+month and adjust permissions ad hoc whenever required due to a person
+joining/leaving the team and/or a project starting/ending.
+
+| System | System Admin |
```

Comment:
How about Heroku?

Reply:
Good point, thanks @vitaliiprodan I've added Heroku and made Ilya Lozer the System Admin. I've also added Deputy System Admins that should have admin permission so they can handle urgent matters while the System Admin is absent (holiday/sick etc).